README.md

```diff
@@ -54,3 +54,33 @@ Collecting multiple `FormulateInput` elements within a single `FormulateForm` el
 ### Comprehensive Documentation
 
 There's a lot more available to read at the comprehensive [documentation website](https://vueformulate.com).
+
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/wearebraid/vue-formulate/graphs/contributors"><img src="https://opencollective.com/vue-formulate/contributors.svg?width=890&button=false" /></a>
+
+### Financial Contributors
```

Comment:
Do we want to list these headings about financial contributors and such? Is there a willingness to accept sponsorship dollars?